draft-polli-ratelimit-headers.md

```diff
@@ -398,7 +398,8 @@ A server MAY return `RateLimit` response header fields independently
 of the response status code.  This includes throttled responses.
 
 If a response contains both the `Retry-After` and the `RateLimit-Reset` header fields,
-the value of `RateLimit-Reset` MUST be consistent with the one of `Retry-After`.
+the value of `RateLimit-Reset` MAY reference the same point in time as
+`Retry-After`.
```

Comment:
I agree, but It could appear to someone reading this that we forgot about the `Retry-After` timestamp when both headers are present. IOW, it is asserting we've looked into it and we add no requirement.

Reply:
Let's `SHOULD` then :P

Reply:
resolved :+1: